crates/trigger-http/src/lib.rs

```diff
@@ -336,6 +338,22 @@ impl HttpTrigger {
                     TokioIo::new(stream),
                     service_fn(move |request| {
                         let self_ = self_.clone();
+                        let span = tracing::info_span!(
+                            "handle_http_request",
```

Comment:
Do we have a convention on where these names come from?  I've seen a few go by and not been quite sure how they relate to function names or discrete activities within a function.  It's tricky I know because something like this is a lambda and has no name of its own...

Reply:
Not really 🤷. Ryan has already raised the great point that we should have some kind of convention around these span names. Right now the best we have is something along the lines of:

> Make sure a span name can be understood without any additional surrounding context e.g. `generate_local_llm_inference` rather than `infer`.

I'm hoping to bake this into the SIP I wrote.